atd-etl/cr3_download/cr3_download.py

```diff
@@ -5,145 +5,186 @@
 
 Description: This script allows the user to log in to the CRIS website
 and download CR3 pdf files as needed. The list of CR3 files to be downloaded
-is obtained from Hasura, and it is contingent to records that do not have
+is obtained from Hasura, and it is made up of the records that do not have
 any CR3 files associated.
 """
 
 import os
 import time
-import json
+import argparse
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
i think this is a great choice for this use case. [asyncio](https://docs.python.org/3/library/asyncio.html) would probably add even more zippiness, but it would introduce substantial refactoring (and 3rd party libs) to get all the `async` operations working 👍 🚢

Reply:
Total agreement from me. This started as a very "quick'n'dirty" hack, and since it was totally a non-requested change, I was going for whatever was quickest. I boxed my time to only what I could reasonably expect to gain back by making the changes for the 30K downloads we had queued up. Since this seems to be working, I'm inclined to agree with your sentiment to leave it be and 🚢